src/bufr/BufrParser/Exports/Transforms/OffsetTransform.h

```diff
@@ -0,0 +1,23 @@
+//
+// Created by Ronald McLaren on 9/24/20.
```

Comment:
Same

Reply:
OBE (Overcome by Events) in a later branch.